packages/facet-filter/README.md

```diff
@@ -54,6 +54,8 @@ import FacetFilter from 'bento-core';
     facetsConfig={facetsConfig}
     CustomFacetSection={CustomFacetSection}
     CustomFacetView={CustomFacetView}
+    enableClearSection // add additional clear button on facet Section
```

Comment:
Hi @rana22, Is this one enabled in bento-RI by default or Out of the box for any new project that are cloning Bento?

Reply:
It should be configured from the projects.